model/clusters_mgmt/v1/aws_infrastructure_access_role_grant_type.model

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an AWS infrastructure access role grant.
+class AWSInfrastructureAccessRoleGrant {
+	// Link to AWS infrastructure access role. 
+	link Role AWSInfrastructureAccessRole
+
+	// The user AWS IAM ARN we want to grant the role.
+	UserARN String
+
+	// State of the grant.
+	State AWSInfrastructureAccessState
+
+	// Description of the state
```

Comment:
End this sentence with a dot. That dot will be used by the tool that generates the documentation to separate sentences. The first sentence will be used as the summary of that generated documentation.